### PR TITLE
RavenDB-8155 New cluster admin permission

### PR DESCRIPTION
**Pull request by iftahbe**
RavenDB-8157 Allow to use two server certificates
RavenDB-8158 Don't allow to delete my own certificate
RavenDB-8159 Endpoint to get permissions by certificate thumbprint

### CODE REVIEW

File: src/Raven.Client/ServerWide/Operations/Certificates/CreateClientCertificateOperation.cs [hunk 1]
```diff
@@ -65,8 +69,11 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                         writer.WritePropertyName("Name");
                         writer.WriteString(_name.ToString());
                         writer.WriteComma();
-                        writer.WritePropertyName("ServerAdmin");
-                        writer.WriteBool(_serverAdmin);
+                        writer.WritePropertyName("operator");
```
Comment (ml054) [inline src/Raven.Client/ServerWide/Operations/Certificates/CreateClientCertificateOperation.cs]: Operator? (upper case).
Why we don't have enum here?

Comment (ml054) [reply src/Raven.Client/ServerWide/Operations/Certificates/CreateClientCertificateOperation.cs]: + maybe use nameof?

File: src/Raven.Client/ServerWide/Operations/Certificates/PutClientCertificateOperation.cs [hunk 2]
```diff
@@ -58,8 +62,11 @@ public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, Serve
                             writer.WritePropertyName("Certificate");
                             writer.WriteString(Convert.ToBase64String(_certificate.Export(X509ContentType.Cert)));
                             writer.WriteComma();
-                            writer.WritePropertyName("ServerAdmin");
-                            writer.WriteBool(_serverAdmin);
+                            writer.WritePropertyName("operator");
```
Comment (ml054) [inline src/Raven.Client/ServerWide/Operations/Certificates/PutClientCertificateOperation.cs]: code duplication? Same here: Operator

File: src/Raven.Client/ServerWide/Operations/Certificates/CreateClientCertificateOperation.cs (hunk 1 above)
Comment (iftahbe) [reply src/Raven.Client/ServerWide/Operations/Certificates/CreateClientCertificateOperation.cs]: Yes, changing to enum

**Pull request closed without merging**